xml/slurm.xml

```diff
@@ -440,22 +439,23 @@ systemctl enable slurmd.service</screen>
       <step>
        <para>
         In preparation of the conversion, make sure the variable
-        <literal>innodb_buffer_size</literal> is set to a value &gt;= 128 Mb:
+        <literal>innodb_buffer_size</literal> is set to a value of 128&nbsp;MB
+        or higher:
```

Comment:
I decided that words were preferable to using &egs; for a proper ≥ symbol. I checked the InnoDB docs and the sizes are in MB (megabytes) not Mb (megabits). MiB might be preferable but I think inconsistent with the rest of the Guide.